apps/web/src/components/SearchRequestTable/SearchRequestTable.tsx

```diff
@@ -29,17 +30,17 @@ const statusAccessor = (
     : "";
 
 function dateAccessor(value: Maybe<string>, intl: IntlShape) {
-  return value ? (
-    <span>
-      {formatDate({
+  return value
+    ? formatDate({
         date: parseDateTimeUtc(value),
         formatString: "PPP p",
         intl,
-      })}
-    </span>
-  ) : null;
+      })
+    : null;
 }
 
+type Cell = IndividualCell<PoolCandidateSearchRequest>;
+
 interface SearchRequestTableProps {
```

Comment:
Related to the other question, I'd expect this to be 
```tsx
type SearchRequestCell = Cell<PoolCandidateSearchRequest>;
```

Reply:
Actually, I should have read both before responding. I like this so I'll update all to match this pattern. Thanks! 😁

Reply:
Thanks for suggestion, done in [simplify table type names](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5649/commits/3c4df795d3588458063b1490824a20c60ba39d6a)